<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetShowProducts</class>
 <widget class="QWidget" name="WidgetShowProducts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Фильтер:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Названия</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Катигоря</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Производите</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ЦЕна</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ПЦена</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Названия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Катигория</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Производитель</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ЦЕна</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ПЦена</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>сохронить в pdf</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
